### ISSUE TITLE
Bug: Set the return value of Max and Min operations to object.

### ISSUE THREAD

**Issue by mikependon** [mikependon/RepoDB#357]
Currently, the implementation is only returning the *long* data type which is a bit limited. We need to make sure that the return type would be of type *object* to support all return types.

**Calls:**

```
connection.Max<Entity>(e => e.DateTime);
connection.Max<Entity>(e => e.Int32);
connection.Max<Entity>(e => e.Double);
```

**Acceptance Criteria:**

All affected *Integration Tests* and *Unit Tests* must be fixed and passing.

**Comment by mikependon** [comment]
The fix will be available at version > v1.10.1-alpha1.

**Comment by mikependon** [comment]
Is now available at RepoDb.MySql (v1.0.1).
